runtime/Java/src/org/antlr/v4/runtime/CharStreams.java

```diff
@@ -303,4 +303,24 @@ public static CodePointCharStream fromChannel(
 			channel.close();
 		}
 	}
+
+	/**
+	 * Takes the stream and forces all symbols to uppercase.
+	 *
+	 * @param in
+	 * @return
+	 */
+	public static CharStream forceUpper(CharStream in) {
```

Comment:
Could we use toUpper perhaps? asUpper?

Reply:
toUpper it is., and changed forceLower to toLower.